### PR TITLE
[MM-673]: Updated the instance connect autocomplete data to show alias and url both

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
#### Summary
Updated the instance connect autocomplete data to show alias and url both

##### Existing
Screenshot from 2024-07-30 19-32-33

##### Updated
Screenshot from 2024-07-30 21-40-48

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-plugin-jira/issues/673

### CODE REVIEW

File: server/instances.go [hunk 1]
```diff
@@ -417,7 +418,7 @@ func (p *Plugin) httpAutocompleteInstalledInstanceWithAlias(w http.ResponseWrite
 	}
 
 	for _, instanceID := range info.Instances.IDs() {
-		item := instances.getAlias(instanceID)
+		item := fmt.Sprintf("%s(%s)", instances.getAlias(instanceID), instanceID)
```
Comment (wiggin77) [inline server/instances.go:421]: Shouldn't there be a space between the alias and url?

Comment (raghavaggarwal2308) [reply server/instances.go:421]: I think we can put the url as a help text

Comment (Kshitij-Katiyar) [reply server/instances.go:421]: Screenshot from 2024-07-30 21-40-48

File: server/command.go [hunk 2]
```diff
@@ -229,7 +229,7 @@ func withParamIssueKey(cmd *model.AutocompleteData) {
 func createConnectCommand() *model.AutocompleteData {
 	connect := model.NewAutocompleteData(
 		"connect", "", "Connect your Mattermost account to your Jira account")
-	connect.AddDynamicListArgument("Jira URL", makeAutocompleteRoute(routeAutocompleteConnect), false)
+	connect.AddDynamicListArgument("Jira URL", makeAutocompleteRoute(routeAutocompleteInstalledInstanceWithAlias), false)
```
Comment (raghavaggarwal2308) [inline server/command.go:232]: Just curious, what is the use of this change?

**Review by raghavaggarwal2308 — APPROVED** [review]

File: server/instances.go (hunk 1 above)
Comment (wiggin77) [reply server/instances.go]: Even better. :+1:

**Review by wiggin77 — APPROVED** [review]

File: server/command.go (hunk 2 above)
Comment (raghavaggarwal2308) [reply server/command.go:232]: Never mind, got it 👍🏽

**Review by raghavaggarwal2308 — APPROVED** [review]

**Review by AayushChaudhary0001 — APPROVED** [review]
This PR has been tested for the following scenarios:-
- Displayed both name and instance link in the suggestion box
- Tested for both cloud and server instance 

The PR was found working fine for both the above conditions, LGTM. Approved.

**Pull request merged**